### ISSUE TITLE
Steam won't log me into my account 

### ISSUE THREAD

**Issue by Crozh** [ValveSoftware/steam-for-linux#10365]
#### Your system information

* Steam client version (build number or date): LATEST and BETA has this same issue
* Distribution (e.g. Ubuntu): Arch KDE 
* Opted into Steam client beta?: [Yes/No] Yes and No
* Have you checked for system updates?: [Yes/No] Yes 
* Steam Logs: 
[steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/13864015/steam-logs.tar.gz)

* GPU: AMD 

#### Please describe your issue in as much detail as possible:
It's been 3 weeks since I've had this issue, steam logs me into my account once every hundred times, I tried reinstalling steam, using another distro (mint), resetting my modem, changing steamapps location, changing to beta, using the flatpak version, making another account, but NOTHING changed.   
Steam on Windows boots fine I think. 
I used -noreactlogin and it was more frequent the times it'd show me the library, instead of buffering infinitely. Maybe that's the problem, I don't know. 

#### Steps for reproducing this issue:

1. Install Steam 
2. Try to log-in 
3. Vid related happens.

**Comment by goocarry** [comment]
same here +1

**Comment by ariSunz** [comment]
yeah, been having this exact same issue for weeks now. also on Arch.
